src/r_things.c

```diff
@@ -512,6 +514,15 @@ void R_ProjectSprite (mobj_t* thing)
   if (abs(tx)>(tz<<2))
     return;
 
+  // [Alaux] Lock crosshair on linetarget
+  if (STRICTMODE(hud_crosshair_target) && thing == linetarget)
+  {
+    crosshair.lockx = FixedMul(tx, xscale / (hires+1)) / FRACUNIT;
```

Comment:
Any reason why you don't use `>> FACBITS` here (and later)?

Reply:
As far as I'm concerned, both `/ FRACUNIT` and `>> FRACBITS` do the exact same thing. I'm not aware of any particular reason to use either of them in a given context, so I prefer to use what I consider to be the most understandable of the two.

Reply:
Not exactly. Bit-shifting a signed integer is "undefined behaviour" whereas disivion by a value is not. Thus the latter requires extra checks by the compiler. However, here we are dealing with screen coordinates (always positive) and so there is no risk we run into undefined territory.

Reply:
Ultimately, do you think it's worth replacing then?

Reply:
It's used in other places where screen coordinates are calculated, so for the sake of consistency, I think yes. Also, `>> hires` is a single bit-shift whereas `/ (hires + 1)` is an addition and a division, so I'd strictly prefer the former. Sorry for being so picky.

Reply:
Oh, don't worry. I'm doing this voluntarily, and I was expecting there to be guidelines to be followed.

Reply:
I know this. I am spending a major part of my free time on these projects (and I love it!) ... 😉